maintainers/maintainer-list.nix

```diff
@@ -21049,6 +21087,11 @@
     github = "viric";
     githubId = 66664;
     name = "Lluís Batlle i Rossell";
+    inactivityReason = ''
+      No new activity since at least 2018-07-22.
+
+      Registered at 2024-06-19 by AndersonTorres.
```

Comment:
Isn't the git history enough for this?

Reply:
Git metadata is not directly accessible to Nix machinery.